python/demo/demo-part1.ipynb

```diff
@@ -4,11 +4,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Dataset in a Nutshell\n",
-    " This demonstrates key functionality and usage of the `dataset` library.\n",
+    "# Scipp in a Nutshell\n",
+    " This demonstrates key functionality and usage of the `scipp` library.\n",
```

Comment:
Minor point: Do you think it would be worth linking to the readthedocs or even give a one-liner about what scipp is? I'm thinking of the cases where the notebooks might be distributed separate to the documentation for scipp.